src/app/cube/library/library.component.html

```diff
@@ -60,10 +60,16 @@ <h2 class='library-items__title'>
                         (titleClicked)='goToItem(item)'
                     ></clark-library-item>
                 </div>
+                <div *ngIf='!libraryItems?.length' class='empty-notification-container'>
```

Comment:
This if statement is nested in the if that states: libraryItems?.length.  Doesn't that mean this is unreachable?

Reply:
We should maybe use more templates too if we can, to break things up easier.

Reply:
I will refactor this today